src/elli_http.erl

```diff
@@ -87,8 +87,8 @@ keepalive_loop(Socket, NumRequests, Buffer, Options, Callback) ->
       Callback  :: elli_handler:callback(),
       ConnToken :: {'keep_alive' | 'close', binary()}.
 handle_request(S, PrevB, Opts, {Mod, Args} = Callback) ->
-    {Method, RawPath, V, B0} = get_request(S, PrevB, Opts, Callback),
     t(request_start),
+    {Method, RawPath, V, B0} = get_request(S, PrevB, Opts, Callback),
```

Comment:
Absolutely! I decided to count request_start right after connection accepted. This way we can account slow lorries.